15-Appendix.Rmd

```diff
@@ -110,6 +110,33 @@ dependencies are exported when you export an `environment.yml` for this project.
 conda env export -n base| grep -v "^prefix: " > /tmp/base.yml && conda env update --prune -n rstudio -f /tmp/base.yml && rm /tmp/base.yml
 ```
 
+#### r-pillar error
+
+If you get an error like this: 
+
+```bash
+Solving environment: failed
+
+ResolvePackageNotFound:
+  - r-pillar==1.4.2=h6115d3f_0
+  ```
+  
+Then the above command needs to be split up. 
+
+1) Run: 
+
+```bash
+conda env export -n base| grep -v "^prefix: " > /tmp/base.yml
+```
+
+2) Delete the r-pillar line from `/tmp/base.yml`. You can do this in RStudio by going to `File >> Open File`, typing in `/tmp`, and then finding the relevant file. 
+
+3) Run the rest of the command: 
+
+```bash
+echo > $CONDA_PREFIX/conda-meta/history && conda env update --prune -n rstudio -f /tmp/base.yml && rm /tmp/base.yml
```

Comment:
What is the `echo > $CONDA_PREFIX/conda-meta/history` part of the command doing?

BTW, thanks a lot for this PR 🌟. We were having a brief chat the other day wondering where we should put these common workarounds. I don't think we reached any conclusion and we were not 100% sure having a big list here was the best solution.

Having said that, maybe having "workarounds" or "gotchas" section in the guidance is good enough until we think about something better? I'm adding @calumabarnett and @r4vi to this as I imagine they may have opinions/suggestions.

Reply:
I have no idea! @r4vi gave me the code originally and it has worked for me and several other people, so he might be able to explain. 

Can see arguments against having a big list in the guidance itself. Do think it's worthwhile having a list somewhere though because the same things come up a lot, and the answers disappear off slack quite fast.

Reply:
FYI: I had a brief chat with Ravi and he said that command is to empty the file content (which makes sense now that I think about it).

Reply:
@xoen I'm very happy to have this info here, right next to the command that has the problem. At some point we might do a knowledge-base with these sorts of things.

Reply:
@davidread 👍 - I think makes sense to have this close to the offending command until we find a better place/idea.